src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -15,7 +15,7 @@
   limitations under the License.
 -->
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-    <application>YOUR_PROJECT_ID</application>
```

Comment:
Uploaded the wrong appe engine xml

Reply:
Good catch. Do you know what the diff is between the two appengine-web.xml docs?

Reply:
The application tag should be our own unique "project id" right?

Reply:
The difference should be the project id? @jess2422